src/python.rs

```diff
@@ -103,6 +103,26 @@ impl<'p> Python<'p> {
     ///
     /// If `globals` is `None`, it defaults to Python module `__main__`.
     /// If `locals` is `None`, it defaults to the value of `globals`.
+    ///
+    /// # Example:
+    /// ```
+    /// use pyo3::{types::{IntoPyDict, PyBytes, PyDict}, prelude::*};
+    /// let gil = pyo3::Python::acquire_gil();
+    /// let py = gil.python();
+    /// let locals = [("ret", py.None())].into_py_dict(py);
```

Comment:
it's probably not needed to initialize "ret" -- the code below would fail anyway on exception since None is not a PyBytes.